src/app/core/config/config-fix.ts

```diff
@@ -1115,7 +1118,6 @@ export const defaultJsonConfig = {
       ["address", "address"],
       ["distance", "privateSchool"],
     ],
-    "showMap": ["address", "address"],
```

Comment:
any chance we can keep the config for map properties in this central place? The usual setup integrates the matching in 3 views, each with (almost) same config so it is really tedious to copy and paste.

Reply:
Architecture wise it is very annoying to have this separate from the sides. Logically, I definitely belongs to the sides (_which properties of this sides entity should be displayed in the app_). So not having it in the side means a lot of back and forth with indices and searching. I would rather not having to do this. Of course we could just assign it at runtime but that just adds unnecessary complexity. I see the issue with the config but it just seems much more logical this way.

Reply:
we could look into setting default configs for the sides in `"appConfig:matching-entities"` (that is currently the other format but sides are also fixed through that).
But at the moment, maybe let's just keep it as you did and come back to this some time later if we have any ideas to simplify code and config.